documentation/packages/README.md

```diff
@@ -8,3 +8,25 @@ description: >-
 # 📦 Packages
 
 Package can be stood up individually using the `instant package init -n <package_name>` command, or they can be included in your own recipes. This can be accomplished by [creating a profile](https://app.gitbook.com/s/TwrbQZir3ZdvejunAFia/getting-started/config#launching-a-profile) that includes the necessary packages and any custom configuration packages.
+
+## Performance Testing
+
+The performance scripts are located in the `test` folder. To run this script against a local or remote server, you can follow these steps:
```

Comment:
The test fodler from where? We need to specify the repository, user won't always have this cloned.